db/migrate/20190609165906_warehouse_case_report.rb

```diff
@@ -0,0 +1,74 @@
+# Denormalised data-warehousing of all case data required to generate
+# Open/Closed reports. There is a one-to-one relationship between
+# this warehouse table and the cases table.
+class WarehouseCaseReport < ActiveRecord::Migration[5.0]
+
+  # Copied from CSVExporter::CSV_COLUMN_HEADINGS (July 2019)
```

Comment:
Worth adding a comment to CSVExporter that if we change them there, they need to be added to a migration to update them in the schema?